README.md

```diff
@@ -1 +1,168 @@
+And there are certainly [other services that compete with Bitly](https://zapier.com/blog/best-url-shorteners/) which may offer you what you want for free (for now).
+
+But with just a little bit of setup, we can accomplish all of this in a much better way.
+Well, at least much better-suited to the _target audience of this approach_: **People/organizations who use GitHub and Git**.
+
+## Comparison
+
+Here's how this approach compares to Bitly and similar services.
+
+**The benefits**:
+
+- Free(er)!
+  You only need to pay for a custom domain name, if you want.
+- Not subject to the pricing whims of large companies.
+  Features and cost shouldn't change.
+- Uses tools and workflows you're already accustomed to and ideally prefer.
+  You don't need to create a new account just for this purpose, like you do for Bitly.
+- You get a nice git history of all of your links; who changed what and when.
+- You're in complete control.
+  With a bit of coding knowledge, you can customize it any way you'd like.
+  Notably, you can use whatever analytics service you want, e.g. Google Analytics.
+
+**The equivalent**:
+
+- Adding/changing/removing links is quick, convenient, and automatic.
+- You can customize the text of your links fully, both the domain and the part after the `/`.
+- You can track analytics for your links.
+- You can restrict who can see and edit the links.
+- You can organize your links and add comments however you want to make maintenance easier.
+- You have to pay if you want a custom domain.
+- People can't use web searches to find private info like Zoom room urls.
+
+**The downsides**:
+
+- More setup.
+- Your redirect lists are not truly 100% hidden from the public[^1].
+- Editing YAML is slightly harder than typing in textboxes, so you could accidentally break the formatting.
+- If things go wrong, you have to troubleshoot it yourself or ask for help.
+
+## How it works
+
+You have a private _redirects_ GitHub repository that contains your redirect lists as [`.yaml`](https://en.wikipedia.org/wiki/YAML) files.
+This is where you specify where you want to redirect from and to.
+You choose who can see or edit these lists using GitHub's permission settings.
+
+You also have a public _website_ GitHub repository that hosts a barebones [GitHub Pages](https://pages.github.com/) website.
+This is what actually performs the redirecting when a user visits a link.
+You can set this website up at a custom domain to make your links shorter and nicer.
+
+After the one-time setup, **all you have to do is edit the `.yaml` files, and everything else updates automatically**, within a minute or so.
+
+<p align="center">
+  <img height="300" src="https://user-images.githubusercontent.com/8326331/197588654-48f065e5-ca42-4ebc-93b0-7096a402ec18.jpg?raw=true" alt="Diagram of this approach. Explained in text form below.">
```

Comment:
Great diagram, helpful! Minor thing: would it make sense to change the title of the top right box to "your-org/redirects" instead of "your-org/redirects-repo" for preciseness? I think a similar change might be possible with "website-repo".

Reply:
I am torn on this. I named and worded things very carefully and consistently in this readme. I wanted to refer to the repos as two distinct names in the readme: _redirects repo_ and _website repo_. I believe I consistently refer to them as such, with the sole exception of the actual repo names themselves. For the repo names, I wanted to call it `redirects-website` instead of just `website` to make sure it was clear that the two repos were related just from their titles.

One solution could be to change all the mentions of _website repo_ to _redirects website repo_, but I feel that that makes it harder to read and makes it easier to confuse the two when reading the readme.

Reply:
Or I could change the diagram to just say e.g. "website repo" instead of `your-org/website-repo`?